src/index.less

```diff
@@ -88,6 +88,13 @@ header .navbar.navbar-static-top {
   tbody > td > dl > dd {
     margin-bottom: 1em;
   }
+
+  .search-result-button {
+      margin: 0;
+      padding: 0;
+      border: 0;
+      background: transparent;
+  }
```

Comment:
https://github.com/NixOS/nixos-homepage/blob/b60ed4af1f3b94b9415312249a8725fbff6ab2aa/packages-explorer/src/styles/app.less#L80-L85

Not sure `text-aiign` is needed here, but `font` and `color` would be. `display` I'm not sure either, might end up being a no-op with bootstrap.

```
	display: inline;
	font: inherit;
	color: inherit;
	text-align: inherit;
```

Why `inherit` font and color? Because form elements in HTML do not use the body's styles by default, but the user-agent/system's "chrome" styles. Let's imagine the user uses a dark theme with white button text, in the past (if it is still not an issue) in this situation the button text color would be white.